webapp/channels/src/components/global_header/right_controls/plan_upgrade_button/index.tsx

```diff
@@ -18,19 +18,6 @@ import WithTooltip from 'components/with_tooltip';
 import {CloudProducts} from 'utils/constants';
 
 const UpgradeButton = styled.button`
```

Comment:
This is no longer needed, so you can substitute `UpgradeButton` for `button` later when used in this component.